src/qonnx/__init__.py

```diff
@@ -1,5 +1,13 @@
-import onnxruntime
+from qonnx import converters  # noqa: F401
```

Comment:
please avoid doing imports at the top-level init here, especially if the tensorflow/QKeras dependency becomes optional, this could break things.